drivers/hmis/app/graphql/types/hmis_schema/household_client.rb

```diff
@@ -0,0 +1,20 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::HouseholdClient < Types::BaseObject
+    description 'HUD Client within a Household'
+    field :id, ID, null: false
+    field :relationship_to_ho_h, String, null: false
+    field :client, HmisSchema::Client, null: false
```

Comment:
We need to be able to resolve the Enrollment on (non-HoH) household clients somehow, because we want to list out the entry/exit dates of all household members (see: household table in the figma mocks).


This might be tricky because It's recurring on itself.

I think we need something like this:

```
{
  enrollment(id: "6558") {
    id
    household {
      householdClients {
        id
        relationshipToHoH
        client {
          id
          firstName
          lastName
        }
        enrollment {
          id
          entryDate
          exitDate
          moveInDate
        }
      }
    }
  }
}
```